MzmlReader/Run.cs

```diff
@@ -27,6 +27,8 @@ public Run()
         public String FilePropertiesAccession;
         public String iRTpath;
         public List<IRTPeak> IRTPeaks { get; set; }
+        public double RunLowerBoundary { get; set; }
```

Comment:
Is this mz boundaries? Is this MS1 or MS2? If it's MS2, we have this information already stored.